package.json

```diff
@@ -40,14 +41,15 @@
     "loader.js": "^4.2.3"
   },
   "engines": {
-    "node": ">= 0.12.0"
+    "node": ">= 4"
   },
   "private": true,
   "dependencies": {
     "broccoli-merge-trees": "^2.0.0",
     "bulma": "^0.3.1",
+    "d3-flame-graphs": "^3.1.0",
```

Comment:
it's using CSS from this package, after my PR to this library is merged I will remove the vendored copy completely